### DISCUSSION TITLE
Parallel through the C interface

### DISCUSSION THREAD

**Discussion by franciscogaluppo** [ERGO-Code/HiGHS#1803] [Q&A] [answered]
How can I enable parallelism through the C interface? Is it possible?

**Comment by jajhall** [comment]
If you're solving a MIP then parallel is the default. 

For an LP, set the parallel option to "On". However, there's little performance gain. 

For a QP there's no parallelism

**Reply by franciscogaluppo** [reply]
But how do I set the parallel option to "On"?

**Reply by jajhall** [reply, marked as answer]
You call [Highs_setStringOptionValue](https://ergo-code.github.io/HiGHS/dev/interfaces/c_api/#Highs_setStringOptionValue-Tuple{Any,%20Any,%20Any}), passing `"parallel"` as `option`, and `"on"` as `value`.
